docs/ens_overview.rst

```diff
@@ -11,11 +11,15 @@ The :mod:`ens` module is included with web3.py. It provides an interface to look
 domains and addresses, add resolver records, or get and set metadata.
 
 
-.. note:: ENSIP-15 introduced a new standard for ENS name normalization. This standard
-    is implemented in web3.py via the ``ensip15_normalization`` flag on the ``ENS`` /
-    ``AsyncENS`` class. It is also available as a flag on relevant ``ens.utils.py``
-    utility methods. For more information, refer to the :ref:`ensip15_normalization`
-    section.
+.. note::
+
+    web3.py ``v6.1.0`` introduced ENS name normalization standard
+    `ENSIP-15 <https://docs.ens.domains/ens-improvement-proposals/ensip-15-normalization-standard>`_
+    by default. This update to ENS name validation and normalization won't affect ~99%
```

Comment:
nit: "by default" might imply that it could be turned off if the user wanted to. Since it's just how things are normalized now, we could leave it off. But up to you!